test.sh

```diff
@@ -0,0 +1 @@
+#./bin/robot report -i $ONT --output results/report_uberon.tsv
```

Comment:
I think this file was mistakenly added. Can you remove it? Thanks!

Reply:
Yep oops.